### ISSUE TITLE
Training strategy

### ISSUE THREAD

**Issue by bo-miao** [lxa9867/R2VOS#1]
Hi,

May I have a few questions about the training process?

In the paper, you mentioned, "The model is trained for 6 epochs with a learning rate
multiplier of 0.1 at the 3rd and the 5th epoch". May I know do you also pre-train coco for 6 epochs with 5 frames or just follow the strategy of Referformer, i.e., 12 epochs with 1 frame?

Thank you.

**Comment by lxa9867** [comment]
Hi,

Thank you for your interest. We pretrain the network on refcoco for 12 epochs with 1 frame and select the model with the highest performance as our pretrained model. Actually, we also tried to pretrain the network with augmented coco (5 frames), the performance doesn't change a lot.

**Comment by bo-miao** [comment]
I see. Thanks for your interesting work.

**Comment by bo-miao** [comment]
Hi,

May I know the performance you got after pretraining on COCO for reference?

**Comment by lxa9867** [comment]
Our method is pretrained on coco, you may refer to the result in the paper.

**Comment by lxa9867** [comment]
Please note that we do not joint train the refcoco and ref-youtube-vos.

**Comment by bo-miao** [comment]
Sorry, I did not find the results of models that were "only" trained on coco in the paper. 

Maybe my question is not very clear. I just wonder about the performance you got that only trained on coco, i.e., before fine-tuning on Ref-YTB, to check if the results I get are reasonable.

**Comment by lxa9867** [comment]
The domain gap between refcoco and refyoutube is not that big but the object classes are not fully overlapped. It seems not that meaningful to conduct this comparison. In this way, in this step, we do not have plan to conduct this experiment. Thank you for your understanding.

**Comment by bo-miao** [comment]
I see. Thank you for your timely reply.
